configs/containers/docker-ubuntu-intel-impi.yaml

```diff
@@ -75,6 +75,11 @@ spack:
       externals:
       - spec: mysql@8.0.32
         prefix: /usr
+    py-pynacl:
+      buildable: false
+      externals:
+      - spec: py-pynacl@1.5.0
+        prefix: /usr/lib/python3/dist-packages/nacl
```

Comment:
@climbfuji  - I set this prefix using the output from `apt-file list python3-nacl`. Is this the right approach or should I have just used "/usr/lib/python3" as a catchall.

I guess I'm just not sure what the prefix is doing for spack.

Reply:
As long as the dependencies build correctly, then you got it right!